src/nupic/algorithms/Connections.hpp

```diff
@@ -186,8 +186,16 @@ class Connections : public Serializable
    * @param numCells           Number of cells.
    * @param connectedThreshold Permanence threshold for synapses connecting or
    *                           disconnecting.
+   *
+   * @params timeseries - Optional, default false.  If true AdaptSegment will not
+   * apply the same learning update to a synapse on consequetive cycles, because
+   * then staring at the same object for too long will mess up the synapses.
```

Comment:
"hen staring at the same object for too long will mess up the synapses."

be more descriptive here? :stuck_out_tongue: